spec/spec_helper_acceptance.rb

```diff
@@ -29,6 +25,10 @@
   c.formatter = :documentation
 
   c.before :suite do
-    copy_fixture_modules_to( hosts )
+    copy_fixture_modules_to(hosts)
   end
 end
+
+configure_beaker do |host|
+  install_package(host, 'grub2')
+end
```

Comment:
This is mixing two sets of helpers (voxpupuli-acceptance and simp_beaker_helpers), which breaks things. You should probably choose one or the other.

Reply:
I'm not following? `#install_package` is core: https://github.com/voxpupuli/beaker/blob/09a424be95c50d573f1911b43aed9ffb7cef11dc/lib/beaker/host/unix/pkg.rb#L86

Reply:
Aw, I see that you pulled in `simp-beaker-helpers`. I'll let you have the ball.

Reply:
`configure_beaker` comes from voxpupuli-acceptance and does more. I debated converting the module completely, but decided to keep it simple for now.